plugins/askrene/askrene.c

```diff
@@ -409,10 +535,17 @@ static const char *get_routes(const tal_t *ctx,
 			rh->delay = delay;
 		}
 		(*amounts)[i] = flows[i]->delivers;
+		struct amount_msat fee;
+		if (!amount_msat_sub(&fee, r->hops[0].amount, flows[i]->delivers))
+			abort();
+		rq_log(tmpctx, rq, LOG_INFORM, "Flow %zu/%zu: %s",
+		       i, tal_count(flows),
+		       fmt_route(tmpctx, r, (*amounts)[i], finalcltv));
```

Comment:
`fee` is not used here, maybe you wanted to show fees in the logs?

Reply:
Oops, part of my debugging before I decided to log the whole thing!  Removed, thanks!